doc/user/content/third-party/debezium.md

```diff
@@ -6,28 +6,25 @@ menu:
     parent: 'third-party'
 ---
 
-[Debezium](https://debezium.io/) works with Materialize in two capacities:
+You can use [Debezium](https://debezium.io/) to propagate Change Data Capture (CDC) data from a database to Materialize, for example MySQL or PostgreSQL.
 
-- Debezium adds change data capture (CDC) to MySQL or PostgreSQL databases,
-  which lets you propagate data from your database to Materialize
-- Debezium's CDC data (known as its envelope) is formatted in a way that
-  expresses arbitrary operations (e.g. updates and deletes) on the database's
-  data. This envelope is a powerful feature that lets Materialize perform more
-  complex analysis by understanding how upstream data is changing.
+Debezium emits records using an envelope that contains valuable information about the change captured, like the `before` and `after` values for each record. This envelope is a powerful structure that lets Materialize perform more complex analysis to understand all CRUD-like operations happening in the upstream database. For more details on CDC support in Materialize, check the [documentation](https://materialize.com/docs/sql/create-source/avro-kafka/#debezium-envelope-details).
 
-For the best experience using Debezium, we recommend following the guidelines
-outlined here.
 
-### PostgreSQL settings
+{{< note >}}
+Currently, Materialize only supports Avro-encoded Debezium records. If you're interested in JSON support, please reach out in the community Slack or leave a comment in [this GitHub issue](https://github.com/MaterializeInc/materialize/issues/5231).
+{{</ note >}}
 
-Materialize relies on Debezium's CDC envelope (which expresses changes to data)
-to understand all CRUD-like operations happening in the upstream database.
+### Database settings
 
-For Debezium to emit the envelope Materialize expects when using PostgreSQL, the
-tables which you're performing CDC on must have the parameter `REPLICA IDENTITY
-FULL` set.
+For the best CDC experience, we recommend following the guidelines in the Debezium documentation for each upstream database:
 
-For example:
+| Database | Settings |
```

Comment:
nit: making this table seems superfluous, the left-hand column doesn't express anything not duplicated in the right-hand column

Reply:
Agreed. I debated about this one for a bit, so goos to have a fresh perspective! What I wanted to do was provide a quick way for users to scan what source databases are well-supported and tested (assuming the list will grow in the future once we _e.g._ support MongoDB CDC). 

That was my general goal with the page update, just make sure that users can get all info about DBZ support in Materialize in the most immediate way possible, because otherwise it's very hard to understand the state of affairs from different bits and pieces in the docs.